spec/samples/osm/exampleModel.osm

```diff
@@ -150,7 +150,7 @@ OS:ScheduleTypeLimits,
   Temperature,                            !- Name
   -60,                                    !- Lower Limit Value
   200,                                    !- Upper Limit Value
-  CONTINUOUS,                             !- Numeric Type
+  Continuous,                             !- Numeric Type
```

Comment:
This was giving a validation error in the HB schema, maybe I should add a check for this.